styx-scheduler-service/src/test/java/com/spotify/styx/flyte/FlyteAdminClientRunnerTest.java

```diff
@@ -283,6 +284,25 @@ public void testEmitFlyteEventsExceptionHandling() throws IsClosedException {
                 Execution.WorkflowExecution.Phase.SUCCEEDED).build()).build(), runState));
   }
 
+  @Test
+  public void testUndefinedShouldNotThrowException() throws Exception {
```

Comment:
could name this something else, testUndefinedShouldDoNothing or testUndefinedShouldPollAgain, but I am fine with naming also.